Source/Blazorise/Components/Collapse/CollapseHeader.razor.cs

```diff
@@ -29,11 +29,12 @@ protected override void BuildClasses( ClassBuilder builder )
         /// Handles the header onclick event.
         /// </summary>
         /// <returns>A task that represents the asynchronous operation.</returns>
-        protected Task ClickHandler()
+        protected async Task ClickHandler()
         {
-            ParentCollapse.Toggle();
+            if ( ParentCollapse != null )
+                await ParentCollapse.Toggle();
 
-            return Clicked.InvokeAsync( null );
+            await Clicked.InvokeAsync( null );
```

Comment:
I think you've removed the explicit null argument on previous examples... 
`await Clicked.InvokeAsync( null );`

Reply:
Yes. In olded blazor versions there was no parameterless InvokeAsync so null was needed. But now we can safely remove it.